.dockerignore

```diff
@@ -1 +1,3 @@
-node_modules/
\ No newline at end of file
+**/node_modules
+embeddable-say-editor/dist
+test-app/dist
```

Comment:
We should probably add the `embeddable-say-editor/ember-build` path here.

Reply:
Good point, I'll add that.